lib/coverage.js

```diff
@@ -34,12 +55,27 @@ internals.prime = function () {
     };
 };
 
-internals.prime();
-
 
 exports.instrument = function (options) {
 
     internals.patterns.unshift(internals.pattern(options));
+
+    if (options.transform) {
```

Comment:
Might be better off doing `Array.isArray(options.transform)` since you are calling `forEach` on it.